artistools/inputmodel/maptogrid.py

```diff
@@ -455,6 +486,7 @@ def main(args: argparse.Namespace | None = None, argsraw: t.Sequence[str] | None
 
     maptogrid(
         ejectasnapshotpath=ejectasnapshotpath,
+        args=args,
```

Comment:
I've been avoiding passing args around in new code, and trying to gradually replace the old code that does this. It makes the function difficult to call, test, and to understand which args are being checked.